<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>role_edit</class>
 <widget class="QWidget" name="role_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{background-color: rgb(255, 125, 184);}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>人物外观</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>161</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="MyLabel" name="label">
       <property name="styleSheet">
        <string notr="true">border: 2px solid #000</string>
       </property>
       <property name="text">
        <string>左面</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="MyLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">border: 2px solid #000</string>
       </property>
       <property name="text">
        <string>反面</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="MyLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">border: 2px solid #000</string>
       </property>
       <property name="text">
        <string>右面</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="MyLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">border: 2px solid #000</string>
       </property>
       <property name="text">
        <string>正面</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>331</width>
      <height>211</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="rowCount">
     <number>7</number>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>属性类型</string>
     </property>
     <property name="background">
      <color>
       <red>181</red>
       <green>181</green>
       <blue>181</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>初始值</string>
     </property>
     <property name="background">
      <color>
       <red>56</red>
       <green>181</green>
       <blue>181</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>每次升级增加</string>
     </property>
     <property name="background">
      <color>
       <red>3</red>
       <green>105</green>
       <blue>182</blue>
      </color>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>属性设置</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>195</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>增加行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>删除行</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>540</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_saveasRole">
       <property name="text">
        <string>保存为人物模板</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_saveasNpc">
       <property name="text">
        <string>保存为npc模板</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>技能添加</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>541</width>
      <height>171</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>技能名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>技能公式</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>技能描述</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>冷却时间</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string notr="true">普通攻击</string>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>188</red>
        <green>204</green>
        <blue>131</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>510</y>
      <width>195</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>增加行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>删除行</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>role_edit</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
